packages/tanstack-solid-query/test/useSubscription.test.tsx

```
@@ -0,0 +1,485 @@
+};
+
+/**
+ * Pause logging until the promise resolves or throws
+ */
+export const suppressLogsUntil = async (fn: () => Promise<void>) => {
+  const _ = suppressLogs();
+
+  try {
+    await fn();
+  } finally {
+    _[Symbol.dispose]();
```

ESLint

[error] Usage of Symbol.dispose is not allowed - use `makeResource()`

(no-restricted-syntax)